### ISSUE TITLE
[react/jsx-curly-brace-presence] False positive on jsx passed to prop

### ISSUE THREAD

**Issue by bhollis** [yannickcr/eslint-plugin-react#3228]
```
return (
  <CollapsibleTitle
    extra={<span className="activity-type">{activity.type}</span>}
  />
);
```

```
"react/jsx-curly-brace-presence": ["error", "never"],
```

After upgrading from 7.28.0 to 7.29.2, this highlights that the `extra` prop should not have curly braces, when it is not possible for them to be omitted.

**Comment by ljharb** [comment]
It is in fact possible for them to be omitted; that's what the `propElementValues` config (added recently) is for.

however, using just "never" should result in `propElementValues` being set to "ignore", so you shouldn't see a warning here.

**Comment by bhollis** [comment]
Ah, interesting. It's certainly not allowed by TypeScript.

**Comment by ljharb** [comment]
Yep, the TS eslint parser doesn't support it, but eslint, babel, and the jsx spec do.
